src/App.js

```diff
@@ -1,24 +1,180 @@
+  // console.log(restOfWeek);
+
   return (
-    <div className="container p-3">
-      <h1 className="mb-5">Schedule</h1>
-      <div className="row">
-        <div className="col-lg-6 col-md-8">
-          <h2 className="fs-6">Showing all</h2>
-
-          <ol className="list-unstyled">
-            {activities.map((activity) => (
-              <li className="hover-light-bg p-2 rounded-2" key={activity.id}>
-                <header>{activity.title}</header>
-                <small className="text-muted">
-                  {dayjs(activity.startTime).format("D MMM [at] H:mm")} • {activity.instructor}
-                </small>
-              </li>
-            ))}
-          </ol>
+    <div className='container p-3'>
+      <h1 className='mb-5'>Schedule</h1>
+      <div className='row'>
+        <div className='col-lg-6 col-md-8'>
+          <h2 className='fs-6'>Showing all</h2>
+          <ul className='list-unstyled'>
+            <h5>Today</h5>
+            {today &&
+              today.map((activity) => {
+                return (
+                  <li
+                    className='hover-light-bg p-2 rounded-2'
+                    key={activity.id}
+                  >
+                    <header>{activity.title}</header>
+                    <small className='text-muted'>
+                      {dayjs(activity.startTime).format('D MMM [at] H:mm')} •{' '}
+                      {activity.instructor}
+                    </small>
+                  </li>
+                );
+              })}
+            <hr />
+          </ul>
+          <ul className='list-unstyled'>
+            <h5>Tomorrow</h5>
+            {tomorrow &&
+              tomorrow.map((activity) => {
+                return (
+                  <li
+                    className='hover-light-bg p-2 rounded-2'
+                    key={activity.id}
+                  >
+                    <header>{activity.title}</header>
+                    <small className='text-muted'>
+                      {dayjs(activity.startTime).format('D MMM [at] H:mm')} •{' '}
+                      {activity.instructor}
+                    </small>
+                  </li>
+                );
+              })}
+            <hr />
+          </ul>
+
+          <ul className='list-unstyled'>
+            {restOfWeek &&
+              restOfWeek.map((activity) => {
+                return (
+                  <div key={activity.id}>
+                    <h5>
+                      {dayjs(activity.startTime).format('ddd[,] MMMM DD')}
+                    </h5>
+                    <li className='hover-light-bg p-2 rounded-2'>
+                      <header>{activity.title}</header>
+                      <small className='text-muted'>
+                        {dayjs(activity.startTime).format('D MMM [at] H:mm')} •{' '}
+                        {activity.instructor}
+                      </small>
+                    </li>
+                    <hr />
+                  </div>
```

Comment:
Without running it, this looks like, it'll show each activity (in the `restOfWeek`) with a date. So if you have two activities falling under a date, that date will appear twice.

Reply:
oh ok yeah true